esptfa_arima/Authentication/views.py

```
@@ -115,9 +134,67 @@ def create(self, request):
+    # for csv convenience importing
+    @action(detail=False, methods=['post'], permission_classes=[IsAdminUser])
+    def bulk_import_csv(self, request):
+        # get the file from the request
+        file = request.FILES['student_import_file']
+
+        # pass into services
+        try:
+            process_csv_import(file)
+        except DRFValidationError as e:
+            return Response({"Validation Error: ": str(e)}, status=status.HTTP_400_BAD_REQUEST)
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)